definitions/ext-gatsby_build/definition.yml

```diff
@@ -0,0 +1,20 @@
+domain: EXT
+# Derived from the Fastly log ingestion, a datacenter is a Point of Presence
+# https://docs.fastly.com/en/guides/fastly-pop-locations
+type: GATSBY_BUILD
+
+synthesis:
+  name: gatsbySite
+  identifier: gatsbySite
+  encodeIdentifierInGUID: false
+  conditions:
+    - attribute: serviceName
```

Comment:
Do you have a sample of telemetry that would match for these? (I'm interested in seeing if we can find a more specific attribute to identify builds).

Reply:
Ye sure here is one of the spans @srvaroa  https://staging.one.nr/0GbRmvbenjy

Reply:
I get this service.name in that span:

![image](https://user-images.githubusercontent.com/346110/119690646-27d41580-be4a-11eb-8293-ab5f114dd91a.png)

So that would not be matching this rule, right?

Reply:
aha ok ye apologies i'll run another test and refactor those suggestions 😬

Reply:
Ok this is the updated span, so `service.name` and `gatsbySite` should be the best way i think?

Reply:
I've updated but open to your advice @srvaroa :D

Reply:
This telemetry is already associated with an ext-service entity though (from the broad rule I think), so we'd have conflicts there right @srvaroa?

Reply:
Hi @rudouglas , following from @IreneMLC 's comment. We need to support [a rule](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/definitions/ext-service/definition.yml#L118) to match EXT-SERVICE entities from OpenTelemetry data, which is arguably too broad.  The rule that you propose here (which would otherwise be sensible) will be problematic in the current implementation as we'll get two potential matches (EXT-SERVICE vs. EXT-GATSBY_BUILD).

We can make a change in the rule engine to work around this and put less priority on EXT-SERVICE, but this involves adding exceptions to the matching process (with code changes & deploys) so I wanted to check if we have other alternatives before going down that route.

Is there a chance that your telemetry does not send the `service.name` attribute at all (so we'd write the rule matching on a different attribute, and avoid the overlap with EXT-SERVICE)? (I understand this may not be possible if you don't control the integration, but just wanted to check first).

Reply:
Thanks @IreneMLC for your help, that does make sense, the service itself does need a `service.name` in order to function, but I can tag it with anything else we like, so i'm wondering if I used `gatsbyService: true` like in the following example https://staging.one.nr/0znQxNW9GRV or some variation on that. As I can decide what tags we add I assume we can work around this without you having to refactor the `EXT-SERVICE`, unless i'm wildly mistaken :P @srvaroa

Reply:
Having the `service.name` is tricky because it will trigger the EXT-SERVICE rule just by being there, but I actually think we have a 3rd option if you have control over the telemetry, which is setting an `entity.type` attribute to `GATSBY_BUILD`.  This will make the rule engine favour that type before it considers EXT-SERVICE.

The changes would be basically

* Add that `entity.type` with value `GATSBY_BUILD` to your telemetry.
* Update the synthesis rule to match on something else than `service.name`, like:

```
 conditions:
          - attribute: gatsbySite
```

Would that work? (I would like to verify this once we merge the rule, but I think this should give us the desired outcome)

Reply:
ok cool, i tried to set `entity.type` as a tag which didn't seem to overwrite it in NRDB, for context here we are using the new `otlp-native-ingest` container so i'm wondering if that's what's setting the `entity.type`:
https://docs-preview.newrelic.com/docs/otlp-native-ingest/

Reply:
I think that's normal, the GATSBY_BUILD type doesn't exist right now so the match will land in EXT-SERVICE (due to the  `service.name` attributes). 

Let me merge the PR so we can see if this works once the type is present.

Reply:
I noticed the condition is missing update, but I can merge after that.

Reply:
oooh nice @srvaroa i made that change there :D 🤞